src/app/InteractionModelEngine.h

```diff
@@ -289,6 +290,8 @@ class InteractionModelEngine : public Messaging::UnsolicitedMessageHandler,
      */
     uint16_t GetMinGuaranteedSubscriptionsPerFabric() const;
 
+    void OnFabricRemoved(const FabricTable & fabricTable, FabricIndex fabricIndex) override;
```

Comment:
Please document this is coming from FabricTable::Delegate.

Reply:
since the pointer removal is happening inside read client destructor.

Reply:
done, thanks